spec/system/report_post_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "user reports another user", type: :system do
+    let!(:user) { create(:user)}
+    let!(:post) {create(:post)}
+
+    it "report a post" do
+        sign_in user
+        visit posts_path
+
+        click_link(href: "/posts/#{post.id}/report_post/new")
+
+        choose ("reason_false_information")
+        click_on "SUBMIT REPORT"
+
+        expect("Report User Successfully.").to be_present

```

Comment:
Sorry, if late ko lang napansin but instead of this expectation, try this:

`expect(page).to have_text("any_flash_message")`

Reply:
If you tried to use yang to be_present na expectation, you type any string and it will still pass. That's why I also changed yung previous test mo and that's on me for not checking thoroughly.

As you can see, I edited your expectation in your previous test. 
![image](https://user-images.githubusercontent.com/94411241/184100781-6345710c-17ea-4ab5-a144-67329d30aa17.png)

And because I changed the flash message content, I saw a failing test, which may look like this: 
![image](https://user-images.githubusercontent.com/94411241/184100945-c6ee7ebe-6a03-4b60-8dc4-e277afad3af3.png)

Reply:
Sorry na po, my bad hehe ! Pacheck po ako ulit kung goods na. Thank you po :)